src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Ui {
+    private final String GREETING = "Hello";
+    private final String BYE = "Goodbye";
+    private Parser parser;
+
+    public Ui() {
+        this.parser = new Parser();
+    }
+
+    public void doGreeting() {
```

Comment:
```suggestion
    public void printGreeting() {
```
Perhaps this function could have been named better? `doGreeting` does not really make grammatical sense. (Same for `doBye`)